### PR TITLE
transformJsonapiAttrs

### PR DESCRIPTION
**Pull request by onumossn**

### CODE REVIEW

**Comment by jnaglick** [comment]
could you demonstrate how this would be used to address to-many deletion|disassociation / destruction issues?

**Comment by onumossn** [comment]
For our case, we can add relationship ids for associations and disassociations in the attributes hash. This would let us let vanilla rails ActiveRecord deal with associations and disassociations. This would remove the need for "displayed" computed properties and any form of marking for associations and disassociations. It doesn't do anything that would require deletions yet.

**Comment by jnaglick** [comment]
@onumossn could you show a code example of what you mean (like, current way vs. new way using this)

**Comment by onumossn** [comment]
```
function(attrs) {
  return {
    ...attrs,
    rel_ids: this.get('rels').map(v => v.id)
  }
}
```

Previously
```
parentModel({
  children: hasMany('child'),
  displayedChildren: filterBy('children', 'markedForDestruction')
})

childModel({
  parent: belongsTo('parent')
})

component({
  parent: parent.create(),
  onDisassociate(child) {
    child.markForDeletion();
    // OR
    this.get('parent').markManyToManyDeletion('children', child);
  } 
})
```

Using the id attributes
```
parentModel({
  children: hasMany('child'),
  displayedChildren: filterBy('children', 'markedForDestruction')
})

childModel({
  parent: belongsTo('parent')
})


component({
  parent: parent.create(),
  onDisassociate(child) {
    this.get('parent.children').removeObject(child);
    // OR
    child.set('parent', null);
  } 
})
```

This way we can choose when we need jsonapi instead of complicating our workflow by using it everywhere when we don't need it.

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "ember-data-jsonapi-extensions",
-  "version": "0.6.1",
+  "version": "0.7.0",
```
Comment (sh9189) [inline package.json]: Hmm.. weird, I see `0.7.0` on master right now https://github.com/jsonapi-suite/ember-data-extensions/blob/master/package.json#L3

**Pull request merged**